docker/runtime-builder/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
+
+# Package versions.
+ARG GO_VERSION=1.16.4
+ARG RUST_NIGHTLY_VERSION=2021-05-07
+ARG OASIS_CORE_VERSION=21.2.7
```

Comment:
Can you remove default value for `OASIS_CORE_VERSION` and require passing it in as build argument.

In the github action you can then set it as we do it in the benchmarks workflow: https://github.com/oasisprotocol/oasis-sdk/blob/e7f68932321cff1ff0304839630784e7b6f1fccc/.github/workflows/docker.yml#L76-L77

(maybe also extract `determine_oasis_core_version` as you did for determining tag).

So that there's not one more file where the version needs to be updated.

---

Same thing can be done for `RUST_NIGHTLY_VERSION` (can get it automatically by reading https://github.com/oasisprotocol/oasis-sdk/blob/main/rust-toolchain).